bidwire/notifiers/massgov_notifier.py

```diff
@@ -0,0 +1,19 @@
+from .base_notifier import BaseNotifier
+from document import Document
+
+
+class MassGovNotifier(BaseNotifier):
+    def get_site(self):
+        return Document.Site.MASSGOV_EOPSS
+
+    def get_additional_list_text(self, doc):
+        return None
```

Comment:
This might work, but introducing the possibility of returning `None` from this method means that all callers need to be aware and do checks before performing basic operations on this return value. If we want to allow returning `None` from here, we could document that. A simple alternative would be to return an empty string.

(The fact that this is conversation is coming up is for me further evidence that the way in which we are using inheritance here is not a great fit.)